src/components/Selector/MultiSelector.tsx

```diff
@@ -66,7 +66,7 @@ export function MultiSelector<
           aria-invalid={!!get(errors, name)?.message}
           aria-disabled={isDisabled}
           as="div"
-          className={`${button} ${styles.selectorButton} p-1 focus-within:border-gray-d1 focus-within:dark:border-blue-l2`}
+          className={`${button} ${styles.selectorButton} p-1`}
```

Comment:
bug: when selecting countries , focus border now is not visible

(master: focused when selecting countries)
<img width="830" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/fd4d707c-fba1-4ccc-889c-6d6deea50a6d">

Reply:
Fixed https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2568/commits/72381756f65dbb2cff34f58cd45960eb88baad71